nservicebus/upgrades/7to8/message-contracts.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Upgrading message contracts from Version 7 to 8
+summary: Instructions on how to upgrade shared message contracts from NServiceBus version 7 to version 8.
+reviewed: 2022-02-02
+component: Core
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 7
+ - 8
+---
+
+NServiceBus version 8 does not support .NET Standard as a framework target. Therefore, message contract assemblies that target `netstandard2.0` and that reference NServiceBus must be updated. This guidance outlines possible approaches.
+
+Note: When using [unobtrusive mode](https://docs.particular.net/nservicebus/messaging/unobtrusive-mode), the contracts assembly doesn't require a reference to NServiceBus and therefore is not affected. Unobtrusive message contracts can continue to target .NET Standard.
```

Comment:
Isn't it better to put this note as the very first sentence? One starts reading, and if they are using the unobtrusive mode they just skip this guidance.

Reply:
hm yeah good point. Not sure which approach is visually better. Maybe we should also make it a warning or so?

Reply:
I think it's okay here. First sentence sets the stage for the guidance, second one gives you a condition that says read no further